homework_1/task_01/fizzBuzz/fizzBuzz.go

```diff
@@ -0,0 +1,36 @@
+package fizzBuzz
+
+import (
+	"strconv"
+
+	"github.com/pkg/errors"
+)
+
+
+func FizzBuzzPlay(start, end int) ([]string, error) {
```

Comment:
`PlayFizzBuzz` would be a better name in my opinion - first verb and then the object.

Another thing - consider how someone would call this function. It would be something like `fizzbuzz.PlayFizzBuzz` so simply `Play` would be considered a good name here as well. That would make the call to this function look like `fizzbuzz.Play`.